### PR TITLE
Add APIs to get and set the master villageID without auth

### PR DESCRIPTION
**Pull request by babstar93**
Add a new table t_usermastervillagemapping

### CODE REVIEW

File: src/main/java/com/iemr/hwc/controller/wo/UserMasterVillage.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+@CrossOrigin
+@RestController
+@RequestMapping(value = "wo/user")
+public class UserMasterVillage {
+    private Logger logger = LoggerFactory.getLogger(CommonMasterController.class);
+
+    @Autowired
+    private UserServiceImpl userService;
+    @CrossOrigin()
+    @ApiOperation(value = "set master village to a user", consumes = "application/json", produces = "application/json")
```
Comment (roopesh-beehyv) [inline src/main/java/com/iemr/hwc/controller/wo/UserMasterVillage.java:24]: It should be a post method

File: src/main/java/com/iemr/hwc/service/user/UserServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    user.setMasterVillage(districtBranchMapping);
+                    user.setActive(true);
+                    user.setCreatedDate(new Timestamp(System.currentTimeMillis()));
+                    user.setLastModDate(new Timestamp(System.currentTimeMillis()));
+                    user = userMasterVillageRepo.save(user);
+                    if(user!=null){
+                        response = "ok";
+                    }
+                    else{
+                        response = "ko";
```
Comment (roopesh-beehyv) [inline src/main/java/com/iemr/hwc/service/user/UserServiceImpl.java]: what is this ko? change it to reasonable string

Comment (babstar93) [reply src/main/java/com/iemr/hwc/service/user/UserServiceImpl.java]: Change done. It's change to "not_ok" @roopesh-beehyv

File: src/main/java/com/iemr/hwc/controller/wo/UserMasterVillage.java (hunk 1 above)
Comment (babstar93) [reply src/main/java/com/iemr/hwc/controller/wo/UserMasterVillage.java:24]: Change done @roopesh-beehyv

**Review by roopesh-beehyv — APPROVED** [review]

**Pull request merged**